CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [15.2.1](https://github.com/theforeman/puppet-foreman_proxy/tree/15.2.1) (2020-10-15)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy/compare/15.2.0...15.2.1)
+
+**Fixed bugs:**
+
+- Do not set ACLs on DHCP leases directory [\#627](https://github.com/theforeman/puppet-foreman_proxy/pull/627) ([lzap](https://github.com/ekohl))
```

Comment:
I'm hesitant to classify this as a bugfix, which is why I suggested a minor release (15.3.0). Also, it wasn't written by lzap. (same below)

Reply:
The joys of creating changelogs by hand. The commit message for that change reads like it fixes a bug.

Reply:
Setting the ACLs is redundant, but I wouldn't classify it as a bugfix.